test/leak-tester-batch.js

```diff
@@ -11,7 +11,7 @@ let db
 
 function print () {
   if (writeCount % 100 === 0) {
-    if (typeof gc !== 'undefined') gc()
+    if (typeof gc !== 'undefined') gc() // eslint-disable-line
```

Comment:
Does `standard` also complain if we instead use `global.gc`?

Reply:
Checking for `global.gc` works. Another question is if we really need this?

Reply:
> Another question is if we really need this?

For leak testing, yes. If GC kicks in too late, it may lead you to believe there's a leak.

Reply:
When is `gc` defined?

Reply:
If you run it with `node --expose_gc`

Reply:
FYI, you can also add a comment to the top of the file indicating which globals you expect to be defined, like this:

```js
/* global gc */
```

More info here: https://standardjs.com/#i-use-a-library-that-pollutes-the-global-namespace-how-do-i-prevent-variable-is-not-defined-errors